js/customer.js

```diff
@@ -0,0 +1,95 @@
+        // If the name, email and contact number are not valid execute this code
+        /*====================================================================*/
+        }else if (!validName(name) && !validPhoneNumber(phoneNumber) && !validEmail(email)) {
+                    name.style.background = '#f00';
+                    email.style.background = '#f00';
+                    phoneNumber.style.background = '#f00';
+        // If the name and email are valid but phone number not execute this code
+        /*====================================================================*/
+        }else if(validName(name) && !validPhoneNumber(phoneNumber) && validEmail(email)) {
+                    phoneNumber.style.background = '#f00';
+        // If the name and phone number are valid but email not execute this code
+        /*==================================================================*/
+        }else if(validName(name) && validPhoneNumber(phoneNumber) && !validEmail(email)) {           
+                    email.style.background = '#f00';
+        // If the email and phone number are valid but name not execute this code
+        /*==================================================================*/
+        }else  if(!validName(name) && validPhoneNumber(phoneNumber) && validEmail(email)) {
+                    name.style.background = '#f00';
+        // If the name is valid but email and phone number are not execute this code
+        /*==================================================================*/
+        }else if(validName(name) && !validPhoneNumber(phoneNumber) && !validEmail(email)) {
+                    email.style.background = '#f00';
+                    phoneNumber.style.background = '#f00';
+        // If the email is valid but name and phone number are not execute this code
+        /*==================================================================*/
+        } else if(!validName(name) && !validPhoneNumber(phoneNumber) && validEmail(email)) {
+                    name.style.background = '#f00';
+                    phoneNumber.style.background = '#f00';
+        // If the phone number is valid but email and name are not execute this code
+        /*==================================================================*/
+        }else if(!validName(name) && validPhoneNumber(phoneNumber) && !validEmail(email)) {
+                    name.style.background = '#f00';
+                    email.style.background = '#f00';
+        }else {
+                    alert('error');
+                    return false;
+        }
+
+    });
+}, false);
+/*==================== Start AJAX Get Data===================*/
+window.addEventListener('load', function () {
+    const resivedNews = document.getElementById('resivedNews');
+resivedNews.style.display = 'none';
+const showNews = document.querySelector('#showNews');
+const url = "https://private-e99507-kabaros.apiary-mock.com/news";
+var request = new XMLHttpRequest();
```

Comment:
if you use const/let then be consistent with that - don't use **var**
(also there is some indentation issues with this code - format the document)